File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for input
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    // Building a pyramid
    int s = 1;
    for (int h = 1; h <= n; h++)
    {
        // Printing out spaces
        s = h;
        while (s <= n - 1)
        {
            printf(" ");
            s++;
        }
        // Printing out the hashes
        for (int i = 1; i <= h; i++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
# Importing the function from cs50 library
from cs50 import get_int

# Prompting the user for an input till they enter an integer
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Printing out the pyramid on the basis of input given by user
for i in range(height):
    print(" " * (height - (i + 1)) + "#" * (i + 1))
